extension/src/popup/views/ReviewAuth/index.tsx

```diff
@@ -18,6 +18,8 @@ import { getTokenDetails } from "@shared/api/internal";
 import { PunycodedDomain } from "popup/components/PunycodedDomain";
```

Comment:
This file is the only non settings related change so just confirming that this is intended for this PR? Lgtm if it is.

Reply:
ooof - my bad. thanks for catching this! I was testing something with Ledger and this ended up on the branch 🤦 Removing this